data-products/src/main/scala/org/sunbird/analytics/exhaust/collection/BaseCollectionExhaustJob.scala

```diff
@@ -315,12 +314,13 @@ trait BaseCollectionExhaustJob extends BaseReportsJob with IJob with OnDemandExh
       .select(col("user_id").as("userid"), col("consentflag"), col("last_updated_on").as("consentprovideddate"));
   }
 
+  // $COVERAGE-OFF$ Disabling scoverage since logTime function is not in use
```

Comment:
A simple test case can be written right to cover this code?

Reply:
Done